hca/dss/__init__.py

```diff
@@ -33,10 +34,11 @@ class DSSClient(SwaggerClient):
     Client for the Data Storage Service API.
     """
     UPLOAD_BACKOFF_FACTOR = 1.618
+    FILE_NESTING_PARAMETER = [2, 4]
```

Comment:
`FILE_NESTING_PARAMETER` tells the CLI where to put slashes to make a directory structure using the checksum.  There should be a comment about that, the naming is a little cryptic.